### PR TITLE
Business project site

### PR DESCRIPTION
**Pull request by emmahogberg88**
https://weeklyflowers.netlify.app/

### CODE REVIEW

File: code/style.css [hunk 1]
```diff
@@ -1,7 +1,545 @@
...
+.modal-btn {
+  font-family: 'Bebas Neue', cursive;
+  letter-spacing: 1.6px;
+  font-size: 1em;
+  text-align: center;
+  text-transform: uppercase;
+}
+
+.infotext p {
+  font-family: 'Bebas Neue', cursive;
```
Comment (sukiphan97) [inline code/style.css:55]: You have repeatedly used these codes for your h3, button, modal-btn, infortext p. 
 `font-family: "Bebas Neue"`
` line-height: 1.6em`

I would reccommed to create a cusomized class for text styling and add them in HTML tags where you want to have the style. For example:
```
   .bebas-font {
      font-family: 'Bebas Neue', cursive;
   letter-spacing: 1.6px; }
```
   By this way, we can advoid repeating the code.

File: code/style.css [hunk 2]
```diff
@@ -1,7 +1,545 @@
...
+    width: 6em;
+  }
+
+  nav ul {
+    margin: 0;
+    padding: 0.5em 0;
+    list-style: none;
+    }
+    
+  nav ul li a {
```
Comment (sukiphan97) [inline code/style.css:206]: Try naming <a class="nav-link>, so you can reuse the code later. As using many sub-tag, or sub-class may be confusing for the one who reads your code

**Review by sukiphan97 — COMMENTED** [review]
Hi Emma,
  Your code was pretty clean, and I did not have any problem to follow through it. I learned quite many things from there as well. A great part is that you commented out everything and the way you named classes, id were pretty concise.  I left few comments which may be helpful for you, so please check. Overall, the site is beautiful!  Well-done! 💯😃

File: code/style.css (hunk 1 above)
Comment (emmahogberg88) [reply code/style.css:55]: This is smart! I will definitely work more with customized classes ahead, thank you! 😃

File: code/style.css (hunk 2 above)
Comment (emmahogberg88) [reply code/style.css:206]: I really needed this tip. For some reason I messed up with my a-tags and couldn´t find a good way out of it. Your tip will definitely help 👍

File: code/script.js [hunk 3]
```diff
@@ -1,9 +1,35 @@
...
+  const firstName = document.getElementById("firstname").value
+  
+  message.style.display = "block"
+  signup.style.display = "none"
+
+  //grab the first chlid of the message (parent) and change the text content
+  message.firstElementChild.textContent = `Hi ${firstName},`
+  
+  //save data from our form and get the delivery frequency to use in text.
+  const ourFormData = new FormData(e.target)
```
[the remark this answers is not in the document]
Comment (emmahogberg88) [reply code/script.js:22]: Right! That was a good piece of code I found/stole somewhere.. :)

**Pull request closed without merging**